File: MusicEventManagementSystem/MusicEventManagementSystem.API/Models/Event.cs
using MusicEventManagementSystem.Enums;

namespace MusicEventManagementSystem.API.Models
{
    public class Event
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime Interval { get; set; }
        public EventStatus Status { get; set; } 
        public Guid CreatedById { get; set; }
        public int LocationId { get; set; }
        //public Location Location { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
        public DateTime? DeletedAt { get; set; }


    }

}

File: MusicEventManagementSystem/MusicEventManagementSystem.Client/src/frontend/event-organization/types/api/event.ts
import { EventStatus } from '../enums/EventOrganization';

export interface EventResponse {
  id: number;
  name: string;
  description: string;
  eventInterval: Date;
  status: EventStatus;
  createdById: string;
  locationId: number;
  createdAt: Date;
  updatedAt: Date;
  deletedAt?: Date;
  ticketTypes?: number[];
  pricingRules?: number[];
}